WooCommerce/Classes/Analytics/WooAnalyticsEvent+ProductCreationAI.swift

```diff
@@ -32,9 +34,13 @@ extension WooAnalyticsEvent {
                               properties: [Key.value.rawValue: tone.rawValue.lowercased()])
         }
 
-        static func generateDetailsTapped(isFirstAttempt: Bool) -> WooAnalyticsEvent {
-            WooAnalyticsEvent(statName: .productCreationAIGenerateDetailsTapped,
-                              properties: [Key.isFirstAttempt.rawValue: isFirstAttempt])
+        static func generateDetailsTapped(isFirstAttempt: Bool,
+                                          features: String) -> WooAnalyticsEvent {
+            let wordCount = features.split { $0 == " " || $0.isNewline }.count
```

Comment:
Nit: this can be simplified using `components(separatedBy:)`:

```suggestion
            let wordCount = features.components(separatedBy: .whitespacesAndNewlines).count
```

Reply:
Thanks for the suggestion, Huong! Done in https://github.com/woocommerce/woocommerce-ios/pull/13411/commits/2a81f1da0a9a83295b40a35132ee8a9d9822c493